.eslintrc.json

```diff
@@ -1,28 +1,42 @@
 {
   "extends": "next",
-  "plugins": ["unused-imports", "import", "visualize-admin"],
+  "plugins": ["import", "visualize-admin"],
```

Comment:
Just for my understanding: the auto fix still removes unused imports ?

Reply:
yes :)